skills/local/skill.py

```diff
@@ -17,12 +17,64 @@
 
 async def predict(request: QueryRequest) -> QueryOutput:
 
+    """
```

Comment:
this file was originally intended to have the most simple, hardcoded skill for local setups. Now it seems a bit messy. Are these changes needed? It seems this was more for your development?

Reply:
sorry, that was an accident, that was some testing for tweac. Already fixed